documentation/TechnicalDesignDocument.tex

```diff
@@ -96,6 +96,26 @@
     In this section build automation and its constituent components will be discussed.
     How we applied it, the software used and the targets and platforms supported are among a few.
 
+    %------------------------------------------------------------------------------------------------------------------------------------------------------- Continuing Development
+    \section{Continuing Development}
+    This section describes the plan on how to continue with development in the best way.
+    The team worked on quite some user stories from april 9th, to june 15th.
+    The following functionalities are most critical at the moment to start working on:
+    \myparagraph{Instantiating maramafications from a .mar file.}
+    A beginning is made at implementing the .mar file.
+    The .mar file is basically a json file with some agreements on structure.
+    The goal of the .mar file is, whenever the file is loaded in, a maramafication is instantiated.
+    This means that every value from the .mar file is evaluated and constructed into a usable maramafication model in the View.
+    \myparagraph{Communication from Editor to View}
+    The Editor must be able to send its state to the View so the View can visually represent that state.
+    This also requires some work inside the View to make this happen.
+    The reason behind this is to keep the Editor end View separated.
+    This wat the Editor and View have their own goal.
+    \myparagraph{Saving the state of the View inside the Editor}
+    The Editor needs to be able to know the state that the View is in.
```

Comment:
Isn't it the other way around? The editor should contain the state and the view should only display stuff and redirect input to the editor